docs/android-setup.md

```diff
@@ -4,19 +4,20 @@ In order to integrate SMKitUI you need to import the smkitui dependency
 Add on project level `build.gradle`:
  ```groovy
 allprojects {
+  repositories {
+    google()
+    jcenter()
     maven {
-        url ${artifactory_contentUrl}
+      url "${artifactory_contentUrl}"
     }
+  }
 }
 ```
-#### FBJNI
-
-Both React Native and SencyMotion use **fbjni**. For example, the versions for SMKitUI that are used for
-development are:
+#### Import SMKitUI and exclude FBJNI - dont not skip this step
```

Comment:
@SencyCaspit grammar "dont not skip this step" -> "do not skip this step"
you can also use some "warning" icon to emphasis it